be/src/storage/persistent_index.cpp

```diff
@@ -1058,8 +1120,11 @@ class FixedMutableIndex : public MutableIndex {
 
     size_t memory_usage() override { return _map.capacity() * (1 + (KeySize + 3) / 4 * 4 + kIndexValueSize); }
 
+    void set_format_version(uint32_t ver) override { format_version = ver; }
+
 private:
     phmap::flat_hash_map<KeyType, IndexValue, FixedKeyHash<KeySize>> _map;
+    uint32_t format_version = kMutableIndexFormatVersion2;
```

Comment:
Should follow the naming rules.
```suggestion
    uint32_t _format_version = kMutableIndexFormatVersion2;
```